test/run_test

```diff
@@ -634,10 +634,12 @@ $volume_options
 
   echo "  Changing passwords"
 
-  # create separate mounting directory for second container, as selinux does
-  # not allow two containers accesing one mounting directory if mounted with
-  # Z option
-  create_volume_dir || ret=1
+  echo "Kill the previous container and create a new one"
+  local cidfile=$CID_FILE_DIR/"${name}"
```

Comment:
Do you think we could benefit from using the generic [ct_get_cid](https://github.com/sclorg/container-common-scripts/blob/1e82b93382d056dd17b90b8e0e637e5bfb9cbb88/test-lib.sh#L335) function from test-lib here?

Reply:
Yes, it makes sense to me, although it would be better to be done in the whole script to keep it consistent. I'd not combine it to this PR now, but might create a separate PR for this and other functions from test-lib that are duplicated here.

Reply:
Some small refactoring prepared here: https://github.com/sclorg/postgresql-container/pull/545 but this and #544 must be merged before, I built those changes based on the commits here in order to avoid merge conflicts later.

Reply:
Agreed.